### PR TITLE
Completed css-brochure-form exercise.

### PR DESCRIPTION
**Pull request by rachel-lyy**
css-brochure-form

### CODE REVIEW

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    </div>
+    <div class="container">
+      <form action="learnMore">
+        <label for="email">COMPANY EMAIL</label>
+        <div><input type="text" id="email" value="sally@company.co"></div>
+        <label for="name">NAME</label>
+        <div><input type="text" id="name" value="Sally Smith"></div>
+        <label for="city">WHERE ARE YOU HIRING?</label>
+        <div><input type="text" id="city" value="City, Country"></div>
+        <label for="">HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label>
```
Comment (BrettAlbright) [inline css-brochure-form/index.html]: make sure to have the proper `for` attribute!

File: css-brochure-form/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <div>
+          <select name="" id="">
```
Comment (BrettAlbright) [inline css-brochure-form/index.html]: make sure to fill in the attribute values!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
looks good! Just update a couple attribute values!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**